libs/features/georouting/georouting.js

```diff
@@ -129,24 +129,20 @@ async function showModal(details) {
   return getModal(null, { class: 'locale-modal', id: 'locale-modal', content: details, closeEvent: 'closeModal' });
 }
 
-export default async function loadGeoRouting(config, createTag, getMetadata) {
+export default async function loadGeoRouting(config, createTag, getMetadata, geoDetails) {
   const { locale } = config;
 
   const urlLocale = locale.prefix.replace('/', '');
   const storedInter = sessionStorage.getItem('international') || getCookie('international');
   const storedLocale = storedInter === 'us' ? '' : storedInter;
 
-  const resp = await fetch(`${config.contentRoot ?? ''}/georouting.json`);
-  if (!resp.ok) return;
-  const json = await resp.json();
-
-  const urlGeoData = json.data.find((d) => d.prefix === urlLocale);
+  const urlGeoData = geoDetails.data.find((d) => d.prefix === urlLocale);
```

Comment:
would be great to use optional chaining to make sure it doesn't throw an error here